packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -55,8 +55,7 @@ const navContainerStyle = css`
 `;
 
 const menuIconButtonStyle = css`
-  background-color: transparent;
-
+  z-index: 1;
```

Comment:
Defining the background color here was resulting in it overriding the active state. The z-index here is to prevent the popover from displaying beneath the active state of Atlas